starlark/testdata/set.star

```diff
@@ -116,3 +116,27 @@ assert.eq(iter(), [1, 2, 3])
 
 # sets are not indexable
 assert.fails(lambda : x[0], "unhandled.*operation")
+
+# adding and removing
+x.add(4)
```

Comment:
Add `x = set()` before each of these stanzas so that there is no residual effect across them.

Reply:
Starlark doesn't permit this because of global variable reassignment, but I will just create a unique set for each block.